### PR TITLE
Staff Access API changes

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#12863

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: search-api/src/search_api/request_handlers/document_access_request_handler.py [hunk 1]
```diff
@@ -57,10 +58,18 @@ def create_invoice(document_access_request: DocumentAccessRequest, user_jwt: Jwt
...
+            waive_fees = True
         for document in document_access_request.documents:
-            filing_types.append({'filingTypeCode': DOCUMENT_TYPE_TO_FILING_TYPE.get(document.document_type.name)})
+            filing_types.append({
+                'filingTypeCode': DOCUMENT_TYPE_TO_FILING_TYPE.get(document.document_type.name).get(role),
+                'waiveFees': waive_fees})
         payment_response = create_payment(str(document_access_request.account_id), filing_types, user_jwt,
-                                          request_json.get('header', {}))
+                                          header)
+        print(payment_response)
```
Comment (kialj876) [inline search-api/src/search_api/request_handlers/document_access_request_handler.py:72]: forgot to rm this print

**Review by kialj876 — APPROVED** [review]
Looks good to me! Are you also adding the db script/migration in auth for the staff accounts? Once I know which ids to use I'll update the UI

File: search-api/src/search_api/request_handlers/document_access_request_handler.py (hunk 1 above)
Comment (lekshmimallika-aot) [reply search-api/src/search_api/request_handlers/document_access_request_handler.py]: Fixed

**Comment by lekshmimallika-aot** [comment]
> Looks good to me! Are you also adding the db script/migration in auth for the staff accounts? Once I know which ids to use I'll update the UI

Will discuss this.

**Pull request merged**